app/commands/v2/put_content.rb

```diff
@@ -304,8 +307,13 @@ def raise_if_links_is_provided
         )
       end
 
-      def validate_schema
-        SchemaValidator.new(type: :schema).validate(payload.except(:content_id))
+      def schema_validation_errors
+        @_validation_errors ||=
+          begin
```

Comment:
i believe so, yes. otherwise it would assign the output of only the first of the three lines in the begin/end block.